src/Patch/Element.js

```diff
@@ -132,7 +132,7 @@ export default function(internals) {
       const oldValue = Native.Element_getAttribute.call(this, name);
       Native.Element_setAttribute.call(this, name, newValue);
       newValue = Native.Element_getAttribute.call(this, name);
-      internals.attributeChangedCallback(this, name, oldValue, newValue, null);
+      internals.attributeChangedCallback(this, name.toLowerCase(), oldValue, newValue, null);
```

Comment:
There are attributes that do rely on capitalization, for example viewBox on `svg`: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/viewBox If we therefore lowercase all attribute names, we would break `svg.setAttribute('viewBox', '....')`. I think the lowercasing should therefore only be in our callback, as we check it there with `observedAttributes`.

Reply:
I thought `viewBox` was in the SVG namespace, so you would need to use `setAttributeNS` for that one? Although, I've definitely been confused when running into this sort of thing before, so I'll have to double check.

Reply:
Shall we move forward with this PR?